extension/src/popup/components/sendPayment/SendSettings/TransactionFee/index.tsx

```diff
@@ -16,44 +15,13 @@ import {
   saveTransactionFee,
   transactionDataSelector,
 } from "popup/ducks/transactionSubmission";
-import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
 
 import "./styles.scss";
 
-enum NetworkCongestion {
-  LOW = "Low",
-  MEDIUM = "Medium",
-  HIGH = "High",
-}
-
 export const SendSettingsFee = () => {
   const dispatch = useDispatch();
   const { transactionFee } = useSelector(transactionDataSelector);
-  const { networkUrl } = useSelector(settingsNetworkDetailsSelector);
-  const [recommendedFee, setRecommendedFee] = useState("");
-  const [networkCongestion, setNetworkCongestion] = useState("");
-
-  useEffect(() => {
-    (async () => {
-      try {
-        const server = new StellarSdk.Server(networkUrl);
-        const {
-          fee_charged: feeCharged,
-          ledger_capacity_usage: ledgerCapacityUsage,
-        } = await server.feeStats();
-        setRecommendedFee(stroopToXlm(feeCharged.mode).toString());
-        if (ledgerCapacityUsage > 0.5 && ledgerCapacityUsage <= 0.75) {
-          setNetworkCongestion(NetworkCongestion.MEDIUM);
-        } else if (ledgerCapacityUsage > 0.75) {
-          setNetworkCongestion(NetworkCongestion.HIGH);
-        } else {
-          setNetworkCongestion(NetworkCongestion.LOW);
-        }
-      } catch {
-        // use default values
-      }
-    })();
-  }, [networkUrl]);
```

Comment:
moving all this logic into a custom hook so we can use the network congestion/fee info in multiple places